src/VisualStudio/Core/Def/EditorConfigSettings/SettingsEditorFactory.cs

```diff
@@ -21,45 +20,22 @@
 
 namespace Microsoft.VisualStudio.LanguageServices.EditorConfigSettings;
 
-[Export(typeof(SettingsEditorFactory))]
 [Guid(SettingsEditorFactoryGuidString)]
-internal sealed class SettingsEditorFactory : IVsEditorFactory, IVsEditorFactory4, IDisposable
+internal sealed class SettingsEditorFactory() : IVsEditorFactory, IVsEditorFactory4
 {
+    private static SettingsEditorFactory? s_instance;
```

Comment:
Consider just public static readonly Instance = new SettingsEditorFactory() and let the class initializer do it.